src/e2e-test/java/io/cdap/plugin/sfmcsource/actions/SfmcSourcePropertiesPageActions.java

```diff
@@ -53,7 +53,7 @@
 public class SfmcSourcePropertiesPageActions {
   private static final Logger logger = LoggerFactory.getLogger(SfmcSourcePropertiesPageActions.class);
   private static Gson gson = new Gson();
-  private static List<String> bigQueryrows = new ArrayList<>();
+  //private static List<String> bigQueryRows = new ArrayList<>();
```

Comment:
remove commented out line

Reply:
Removed the line, there was build failure I was looking into that, fixed some things re-triggered the build.